### PR TITLE
JHipster blog post

### PR DESCRIPTION
**Pull request by sebadoom**
Full article, RFC.
quick tip easily generate a full-stack java javascript app

### CODE REVIEW

File: _posts/2016-04-05-quicktip-use-jhipster.markdown [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+# In case you picked Gradle:
+gradle bootRun
+```
+
+#### 3. Fire up Your IDE!
+I love [IntelliJ IDEA](https://www.jetbrains.com/idea/). Most Java IDEs can import Maven and Gradle project setups. Open your IDE and point it to your new JHipster project directory.
+
+![Import JHipster project](https://cdn.auth0.com/blog/jhipster/import.png)
+
+Additional IDE specific settings may improve your experience. For instance, you may want to setup autocompletion for the frontend libraries if supported. JHipster docs provide details for [Eclipe and IntelliJ IDEA](http://jhipster.github.io/configuring-ide/).
```
Comment (diegopoza) [inline _posts/2016-04-05-quicktip-use-jhipster.markdown]: Typo -> Eclipse

File: _posts/2016-04-05-quicktip-use-jhipster.markdown [hunk 2]
```diff
@@ -0,0 +1,340 @@
+---
+layout: post
+title: "Quick Tip: Easily Generate a Full-Stack Java + JavaScript App"
```
Comment (chenkie) [inline _posts/2016-04-05-quicktip-use-jhipster.markdown]: Maybe this one shouldn't actually be in the quick tip series, given its length. What do you think?

File: _posts/2016-04-05-quicktip-use-jhipster.markdown [hunk 3]
```diff
@@ -0,0 +1,340 @@
...
+- full-stack
+- yeoman
+- spring
+- angular.js
+- microservice
+- microservices
+- rest
+---
+
+Java's library ecosystem is deep and complex. This is both a blessing and a curse: there are great tools out there, but picking the right combination and setting them up to work together can be quite complex. Furthermore, the depth and breath of many of these libraries makes the getting-started step somewhat hard. [JHipster]() comes to the rescue. See how easy it is to get started on your new Java + JavaScript web app with it. Read on!
```
Comment (chenkie) [inline _posts/2016-04-05-quicktip-use-jhipster.markdown]: JHipster link

File: _posts/2016-04-05-quicktip-use-jhipster.markdown [hunk 4]
```diff
@@ -0,0 +1,340 @@
...
+
+### Modern web apps: REST and Microservices
+The rise of [IaaS](https://en.wikipedia.org/wiki/Infrastructure_as_a_service) and [PaaS](https://en.wikipedia.org/wiki/Platform_as_a_service) solutions has pushed [microservices](https://auth0.com/blog/2015/09/04/an-introduction-to-microservices-part-1/) to the forefront. Familiarity with the web has also made [REST](https://en.wikipedia.org/wiki/Representational_state_transfer) the preferred architectural style. Java's extensive support for all sorts of technologies does not fail in this case. A whole bunch of libraries and frameworks are available to pick from. So what are some of the most popular alternatives?
+
+#### Spring
+For a long time, [Spring](https://spring.io/) has been one of the most used frameworks in the Java world. With a strong inversion-of-control (IoC) ideology and a somewhat pervasive reliance on annotations in recent releases, Spring has uses in nearly any backend application. Spring covers many areas of backend development: authentication/authorization, data access, messaging, transaction management, REST services, testing, among others.
```
Comment (chenkie) [inline _posts/2016-04-05-quicktip-use-jhipster.markdown]: Can we link to a description of IoC?
`Spring has uses in` => `Spring has use cases in`

File: _posts/2016-04-05-quicktip-use-jhipster.markdown [hunk 5]
```diff
@@ -0,0 +1,340 @@
...
+
+#### Netflix Tools
+Netflix has contributed a series of open-source tools based on their necessities. [Eureka](https://github.com/Netflix/eureka/wiki/Eureka-at-a-glance), a service registry and failover service that goes hand-in-hand with microservice-based architectures; [Zuul](https://github.com/Netflix/zuul/wiki), a proxy with dynamic routing; and [Ribbon](https://github.com/Netflix/ribbon/wiki), a load-balancer.
+
+#### Hibernate
+[Hibernate](http://hibernate.org/), an implementation of the Java Persistence API, is a popular alternative for object-relational mapping (ORM). Although you may prefer to skip ORM, Java has great support for it.
+
+#### SQL and NoSQL
+Popular choices in the SQL camp include [MySQL/MariaDB](https://en.wikipedia.org/wiki/MariaDB) and [PostgreSQL](http://www.postgresql.org/). In the MoSQL camp: [MongoDB](https://www.mongodb.org/) and [Cassandra](http://cassandra.apache.org/) are good alternatives.
```
Comment (chenkie) [inline _posts/2016-04-05-quicktip-use-jhipster.markdown]: `MoSQL` as in "MoustacheSQL"? ;)

**Pull request merged**